packages/sirius-web/frontend/sirius-web/package.json

```diff
@@ -26,14 +26,17 @@
     "@eclipse-sirius/sirius-components-trees": "*",
     "@eclipse-sirius/sirius-components-validation": "*",
     "@eclipse-sirius/sirius-web-application": "*",
+    "@floating-ui/dom": "^1.1.1",
+    "@floating-ui/react": "^0.19.0",
```

Comment:
We fixed the version for all the other import, could you do the same for the new ones (remove the ^)